frontend/src/components/layout/SideNavBar/SideNavbar.test.tsx

```diff
@@ -17,7 +16,7 @@ const renderComponent = (props?: IRenderProps) =>
     <TestCommonWrapper
       history={history}
       roles={props?.roles ?? [Roles.REAL_ESTATE_MANAGER]}
-      claims={[Claims.LEASE_VIEW]}
+      claims={[Claims.LEASE_VIEW, Claims.RESEARCH_VIEW]}
```

Comment:
hmm, is this necessary? does the user have the ability to view research files on the sidenavbar?

Reply:
I think it does. The visibility of the link on the side bar is controlled by the claims